model/global/global.go

```diff
@@ -0,0 +1,12 @@
+package global
+
+// Global stores internal global tracking information for each build variant.
+type Global struct {
```

Comment:
Can this be called something less generic? A "global" collection implies this could eventually be used for other global things. That or maybe the collection name could be more specific, like "global_variant_data"

Reply:
Sorry I didn't see that this struct already exists 😭